src/main/java/com/post/parcel_registration/model/Parcel.java

```diff
@@ -0,0 +1,90 @@
+package com.post.parcel_registration.model;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+public class Parcel {
+    @Id
+    @GeneratedValue
+    private Long id;
+    private String name;
+    private int waight;
+    private long idFrom;
```

Comment:
Is not better to use @JoinColumn and use id of sender and recipient as referencedColumnName?
Then you may remove redundant idFrom and idTo fields. Especially taking into account that you used Eager fetch type, for why did you duplicate the same data from sender and recipient into separate fields?